docs/GOVERNANCE.md

```diff
@@ -0,0 +1,139 @@
+- reporting issues on our
+  [GitHub issues page](https://github.com/napari/napari/issues);
+- proposing a change to the documentation, or
+  [tutorials](https://github.com/napari/napari-tutorials) via a
+  GitHub pull request;
+- discussing the design of the napari or its tutorials on in existing
+  [issues](https://github.com/napari/napari/issues) and
+  [pull requests](https://github.com/napari/napari/pulls);
+- discussing examples or use cases on the
+  [image.sc forum](https://forum.image.sc/tags/napari) under the #napari tag; or
+- reviewing [open pull requests](https://github.com/napari/napari/pulls)
+
+among other possibilities. Any community member can become a contributor, and
+all are encouraged to do so. By contributing to the project, community members
+can directly help to shape its future.
+
+Contributors are encouraged to read the [contributing guide](CONTRIBUTING.md).
+
+## Core developers
+
+Core developers are community members that have demonstrated continued
+commitment to the project through ongoing contributions. They
+have shown they can be trusted to maintain napari with care. Becoming a
+core developer allows contributors to merge approved pull requests, cast votes
+for and against merging a pull-request, and be involved in deciding major
+changes to the API, and thereby more easily carry on with their project related
+activities. Core developers appear as organization members on the napari
+[GitHub organization](https://github.com/orgs/napari/people) and are on our
+[@core-devs](https://github.com/orgs/napari/teams/core-devs) GitHub team. Core
+developers are expected to review code contributions while adhering to the
+[core developer guide](CORE_DEV_GUIDE.md). New core developers can be nominated
+by any existing core developer, and for details on that process see our core
+developer guide.
+
+## Steering Committee
+
+The Steering Committee (SC) members are core developers who have additional
+responsibilities to ensure the smooth running of the project. SC members are
+expected to participate in strategic planning, approve changes to the
+governance model, and make decisions about funding granted to the project
+itself. (Funding to community members is theirs to pursue and manage.) The
+purpose of the SC is to ensure smooth progress from the big-picture
+perspective. Changes that impact the full project require analysis informed by
+long experience with both the project and the larger ecosystem. When the core
+developer community (including the SC members) fails to reach such a consensus
+in a reasonable timeframe, the SC is the entity that resolves the issue.
+
+The steering committee is currently fixed in size to N members. This number may be increased in
+the future, but will always be an odd number to ensure a simple majority vote outcome
+is always possible. The initial steering committee of napari consists of
+
+- XXXXXXXXX.
+...
+- XXXXXXXXX.
+
+The SC membership is revisited every January. SC members who do
```

Comment:
what happens when a SC member becomes inactive and votes therefore have the possibility of being tied?

Reply:
we should add language to ensure that the size is maintained at at least 3 if someone wants to step down etc.

Reply:
Also who visits SC membership? SC itself? :)  
PS: my question is just rhetorical, it doesn't need an answer. Just dropped that for future reference. :)

Reply:
have added language about postponing deadlocked votes during the period of time when there is an even number of SC members